conformance/buf.gen.yaml

```diff
@@ -0,0 +1,7 @@
+version: v1
+managed:
+  enabled: true
+  java_package_prefix: "build"
+plugins:
+  - plugin: buf.build/protocolbuffers/java:v23.4
```

Comment:
Same here - I think we should switch to use local generation using buf/protobuf gradle plugins if we can.